src/index.tsx

```diff
@@ -29,9 +29,19 @@ const eventEmitter = new NativeEventEmitter(Membrane);
 export enum ParticipantType {
   Remote = 'Remote',
   Local = 'Local',
-  LocalScreencasting = 'LocalScreencasting',
 }
 
+export enum TrackType {
+  Audio = 'Audio',
+  Video = 'Video',
+}
+
+export type Track = {
+  id: string;
+  type: TrackType;
+  metadata: Metadata;
```

Comment:
Would we benefit from putting here actual track too (the one of type MediaStreamTrack)?

Reply:
I don't think this is a good idea. I don't see any obvious benefits. Actual tracks have many functions and just passing a track to RN side won't work (that's why we have functions that convert the objects to "RN format"). You can't do anything meaningful just with insides of tracks without access to native (kotlin) side. If there is a missing feature, user can ask us to implement it or make a patch.